internal/process_test.go

```diff
@@ -1250,25 +1278,29 @@ func TestFetchProjects(t *testing.T) {
 			}}}
 		exporter := mock_app_export.NewMockExporter(gomock.NewController(t))
 		client := mock_integration_rest.NewMockClient(gomock.NewController(t))
-		client.EXPECT().GetProjects().Return(projects, nil).Times(1)
+		client.EXPECT().GetProjects(gomock.Any(), gomock.Eq(teamName), gomock.Eq(projectsIds), gomock.Eq(0),
```

Comment:
No need for `gomock.Eq(x)` you can just use `x` directly.

Reply:
Ok